galgebra/lt.py

```diff
@@ -154,10 +155,7 @@ class Lt(object):
 
     @staticmethod
     def setup(ga):
-        # coords = [Symbol('mu_' + str(x)) for x in ga.coords]
-        coords = ga.coords
-        x = sum([coords[i] * ga.basis[i] for i in ga.n_range])
```

Comment:
Note that above, `coord_vec` is defined as `sum([coord * base for coord, base in zip(self.coords, self.basis)])`, which is identical